sdk/Sdk.Generators/FunctionMetadataProviderGenerator/FunctionMetadataProviderGenerator.Parser.cs

```diff
@@ -632,7 +632,8 @@ private bool TryGetAttributeProperties(AttributeData attributeData, Location? at
                             }
                             else
                             {
-                                // TODO: Log diagnostic error
+                                _context.ReportDiagnostic(Diagnostic.Create(DiagnosticDescriptors.InvalidBindingAttributeArgument, attribLocation));
```

Comment:
So is this preventing user to explicitly specify null value for a property when using a binding? Is that something really we should throw an error for?

Reply:
Also synced offline. If we don't think we can think of a scenario where this will happen, probably we should not add this change. The less code we ship, the better.